views/signUp.hbs

```diff
@@ -0,0 +1,32 @@
+{{> header}}
+<section class="section-signup">
+    <div class="row">
+        <div class="signup-text">
+            <p>Signup</p>
+            <p>We do not share your details with anyone.</p>
+        </div>
+        <div class="signup-form">
```

Comment:
Incorrect BEM structure. Define generic BEM for forms, and use it across different forms.